commands/cfformat/settings/wizard.cfc

```diff
@@ -0,0 +1,54 @@
+component {
+
+    property name="fileSystemUtil" inject="FileSystem";
+    property name='formatterUtil' inject='formatter';
+
+    function run() {
+        var examples = deserializeJSON(
+            fileRead( expandPath( "/cfformat/data/examples.json" ) )
```

Comment:
It looks to me like `expandPath('/')` defaults to the directory CommandBox was started in? Is this why you added the `this.cfmapping = "cfformat";` to `ModuleConfig.cfc`? In any case, I would prefer to just make use of
```
property name="cfformat" inject="CFFormat@commandbox-cfformat";
``` 
Since cfformat has the examples as a property already you can just call `cfformat.getExamples()`.